app/schemas/models/user.js

```diff
@@ -406,9 +406,9 @@ _.extend(UserSchema.properties, {
   seenPromotions: {
     type: 'object',
     properties: {
-      'hackstack-beta-release-modal': { type: 'boolean' },
-      'curriculum-sidebar-promotion-modal': { type: 'boolean' },
-      'hp-junior-modal': { type: 'boolean' }
+      'hackstack-beta-release-modal': c.date(),
```

Comment:
unless we are planning to backfill data, user.save might start to fail. Good idea to keep both boolean and date

Reply:
@mrfinch fixed.